pandera/checks.py

```diff
@@ -1022,3 +1021,43 @@ def _str_length(series: pd.Series) -> pd.Series:
                 f"str_length({min_value}, {max_value})",
             ),
         )
+
+    @classmethod
+    @st.register_check_strategy(st.isin_strategy)
```

Comment:
I can help create data synth strategy for this, it's not quite the same as the `isin_strategy`